portfolio/src/main/webapp/template.html

```diff
@@ -27,7 +27,58 @@
     
     <!-- Theme CSS -->  
     <link id="theme-style" rel="stylesheet" href="assets/css/styles.css">
+    <script src="https://aframe.io/releases/1.0.4/aframe.min.js"></script>
+    <script src="https://unpkg.com/aframe-environment-component@1.1.0/dist/aframe-environment-component.min.js"></script>
+
+
+    <script>
+      function getRandomArbitrary(min, max){
```

Comment:
Nit:
Arbitrary in getRandomArbitrary does not seem to be adding much to the name. Maybe getRandomBetween?